src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    public ColorSupplier(Random random) {
+    }
```

Comment:
What does exactly this constructor do? I think we can remove it or initialize Random inside